Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md

```diff
@@ -1,93 +1,56 @@
-| self_deployed | Whether to use a self deployed Azure Application. | False |
-| incidentType | The incident type. | False |
-| display_full_email_body | Whether to fetch incidents with the entire email body, or just an email body preview, mark if the full email body is required. | False
-| look_back | Advanced: Time in minutes to look back when fetching emails. | False
-4. Click **Test** to validate the URLs, token, and connection.
-
-### Using National Cloud
-Using a national cloud endpoint is supported by setting the *Server URL* parameter to one of the following options:
-* US Government GCC-High Endpoint: `https://graph.microsoft.us`
-* US Government Department of Defence (DoD) Endpoint: `https://dod-graph.microsoft.us`
-* Microsoft 365 Germany Endpoint: `https://graph.microsoft.de`
-* Microsoft Operated by 21Vianet Endpoint: `https://microsoftgraph.chinacloudapi.cn`
-
-Refer to [Microsoft Integrations - Using National Cloud](https://xsoar.pan.dev/docs/reference/articles/microsoft-integrations---authentication#using-national-cloud) for more information.
-
-## Email Attachments Limitations
-* The maximum attachment size to be sent in an email can be 150-MB. [large-attachments](https://docs.microsoft.com/en-us/graph/outlook-large-attachments?tabs=http)
-* The larger the attachment, the longer it would take for a command that supports adding attachments to run.
-* Requires the permission of Mail.ReadWrite (Application) - to send attachments > 3mb
-* When sending mails with large attachments, it could take up to 5 minutes for the mail to actually be sent.
-
+    | **Parameter** | **Description** | **Required** |
+    | --- |-------| --- |
+    | Server URL | The host URL | True  |
+    | Application ID or Client ID | The app registration ID | False |
+    | Token or Tenant ID | The tenant ID | False |
+    | Key or Client Secret | The app registarion secret | False |
+    | Certificate Thumbprint | Used for certificate authentication. As appears in the "Certificates & secrets" page of the app. | False |
+    | Private Key | Used for certificate authentication. The private key of the registered certificate. | False |
+    | Fetch incidents |  | False |
+    | Email address from which to fetch incidents | For example, "example@demisto.com" | False |
+    | Name of the folder or sub-folder from which to fetch incidents | Supports folder ID and sub-folders, for example Inbox/Phishing. | False |
+    | First fetch timestamp | &lt;number&gt; &lt;time unit&gt;, for example 12 hours, 7 days. | False |
+    | HTTP Timeout | The timeout of the HTTP requests sent to Microsoft Graph API (in seconds). | False |
+    | Maximum number of emails to pull per fetch |  | False |
+    | Trust any certificate (not secure) |  | False |
+    | Use system proxy settings |  | False |
+    | Use a self deployed Azure application | Rather to use a selp-deployed application | False |
+    | Incident type |  | False |
```

Comment:
What should the description for Incident type be? If this is a string for the incident type, whey is the default value false?